SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialNetDriver.h

```diff
@@ -38,10 +38,6 @@ DECLARE_LOG_CATEGORY_EXTERN(LogSpatialOSNetDriver, Log, All);
 DECLARE_STATS_GROUP(TEXT("SpatialNet"), STATGROUP_SpatialNet, STATCAT_Advanced);
 DECLARE_DWORD_ACCUMULATOR_STAT_EXTERN(TEXT("Consider List Size"), STAT_SpatialConsiderList, STATGROUP_SpatialNet,);
 
-DECLARE_EVENT(USpatialNetDriver, FOnConnectedEvent);
-DECLARE_EVENT_OneParam(USpatialNetDriver, FOnDisconnectedEvent, const FString&);
```

Comment:
What happened to this event?

Reply:
moved to gameinstance instead

Reply:
Game instance only has OnConnected and OnConnectionFailed as far as I can see.

Reply:
`OnDisconnectedEvent` was removed by @padhansell in a separate PR, I'm not sure why the changeline for that is included in my pR